docs/edge_migration.md

```diff
@@ -0,0 +1,73 @@
+# Plans for migrating edge traffic to GCP Ingestion
+
+This document outlines plans to migrate edge traffic to go through the code in this repository.
+
+## Current state
+
+Today, data producers send to a data ingestion stack on AWS as described [here](https://github.com/mozilla/firefox-data-docs/blob/042fddcbf27aa5993ee5578224200a3ef65fd7c7/src/concepts/pipeline/data_pipeline_detail.md#ingestion).
+
+## Phase 1
+
+Timeline: **Q4 2018**
+
+```
+Data Producers -> AWS Edge -> Kafka -> AWS Hindsight -> PubSub -> GCP Hindsight w/ HTTP Output -> GCP Edge
+```
+
+In this configuration we get 100% of "expected" data going to the GCP pipeline without any risk of affecting production AWS processing.
+
+This "expected" data does _not_ include recording and replaying traffic that we currently throw away at the edge (e.g. pre-unified telemetry). There may be some other subtle differences from the original incoming data from producers, such as missing some headers that are not currently being stored in Landfill. On the whole, this a good enough approximation of 100% of the data we care actually about, but for the purposes of testing the new system, we will need to care about operation while processing data we don't care about, see more detail in Phase 3 below.
```

Comment:
There are multiple errors here. I know I wrote this and I'm correcting myself, but now that this is going into actual documentation I would suggest:

    On the whole, this is a good approximation of 100% of the data we actually care about. We also need to test operation of the new system while processing data we don't care about; see more detail in Phase 3 below.